app/apphandlers/embeddedAppHandler.go

```diff
@@ -119,7 +125,8 @@ func (a *embeddedApp) StartRudderCore(ctx context.Context, options *app.Options)
 	}
 	defer sourceHandle.Stop()
 
-	reportingI := a.app.Features().Reporting.GetReportingInstance()
+	reportingI := a.app.Features().Reporting.GetReportingInstance(types.Report)
+	errorReportingI := a.app.Features().Reporting.GetReportingInstance(types.ErrorDetailReport)
```

Comment:
[minor] avoid type suffix `I`
```suggestion
	reporting := a.app.Features().Reporting.GetReportingInstance(types.Report)
	errorReporting := a.app.Features().Reporting.GetReportingInstance(types.ErrorDetailReport)
```

Reply:
Made the changes. Resolving this comment